.github/workflows/rust.yml

```diff
@@ -82,7 +82,16 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
-          args: -v
+          args: -v  
+      - name: Run cargo-tarpaulin
+        uses: actions-rs/tarpaulin@v0.1
+        with:
+          version: '0.12.4'
```

Comment:
We don't need to specify a version here. The documentation mentions that if you don't specify anything, you will be getting the latest version.

Reply:
Removed